.github/workflows/periodic-style-checks.yml

```
@@ -0,0 +1,27 @@
+name: Periodic Style Checks
+
+on:
+  schedule:
+    - cron: "0 1 * * 4" # Runs at 01:00 AM on every Wednesday
+
+jobs:
+  vale:
+    name: Style checker
+    runs-on: ubuntu-22.04
+    defaults:
+        run:
+            shell: bash
+            working-directory: .
+    steps:
+      - uses: actions/checkout@v4
+      - name: Run vale on .rst files        
+        run: |
+          make vale TARGET=*.rst > vale_results.txt
+          sed -e 's/\x1b\[[0-9;]*m//g' vale_results.txt > vale_results_clean.txt
```

Is there a reason to only run on RST files?

```suggestion
      - name: Run vale on RST and MD files        
        run: |
          make vale TARGET="*.rst *.md" > vale_results.txt
          sed -e 's/\x1b\[[0-9;]*m//g' vale_results.txt > vale_results_clean.txt
```